test.sh

```diff
@@ -16,7 +16,10 @@
 
 source base.sh
 
-#Test for codeworld-compiler
-run codeworld-compiler cabal configure --enable-tests
-run codeworld-compiler cabal build 
-run codeworld-compiler cabal test
+DirList="codeworld-compiler"
```

Comment:
The problem with base case is, it is resulting in `directory-name.cabal` while run function needs the name of the directory, so I updated it with `cut`. Is it okay?